### ISSUE TITLE
Feature Adjustment

### ISSUE THREAD

**Issue by jayesh-sapkale** [jswalker-mvc/jswalker#5]
New Features ;
 - Prevent Socket throttling on both side
 - Low Memory Footprint
 - Log handler (File is default ,But mysql And MongoDb Support add later)
 - Multi mongo support.
 - Dashboard
 - Mysql database accessible from in any project with jswalker.
 - Remove unnecessary function call from jswalker core and supporting libs.

Propose Date : 
 - #28-July-2016  +0400

Execution Time span :
 - 28-July-2016 +1000 To 3-August-2016 

Testing :
 - 2 Days

Delay : 
 - None 

Release Date : 
- #07-August-2016 +1900

**Comment by jayesh-sapkale** [comment]
Log entry:
31-July-2016 : Dashboard feature is about to finish.

Remaining thing is : Update error log and status.

**Comment by jayesh-sapkale** [comment]
Log entry:
31-July-2016 : Dashboard feature is about to finish.

Remaining thing is : Update error log and status.

New feature added : Backup facility (Full or partial with dashboard)

**Comment by jayesh-sapkale** [comment]
Log Entry :
31-July-2016  : Dashboard finished 

Testing { Everything }

Test list  :  

Dashboard : 

- [ ] Console 
- [ ] Memory snap
- [ ] Automation
- [ ] Error log
- [ ] Backup

Project Architecture : 
- [ ] Controller 
- [ ] Model
- [ ] Views
- [ ] Render
- [ ] Micro-service
- [ ] Static resource
- [ ] Each page static resource easily available
- [ ] Authentication (Strict)
- [ ] SEO 


Shelter 
- [ ] Entry interface  { Strict and useful }
- [ ] Error handling ( Regular , Socket  : File and Mysql }
- [ ] Memory tracing handling  ( Regular , Socket  : File and Mysql }
- [ ] Template engine ( Different flavor with syntax ) 
- [ ] Favicon 
- [ ] Hot-box
- [ ] SSL Support


Grunting : 
- [ ]  Uglify & Minify 

Package.json
- [ ] Necessary NPM 
- [ ] Validate standard
 
 
Test Result possible date :  3-August-2016 +1900

**Comment by jayesh-sapkale** [comment]
Log Entry :
1-August-2016  +1322

New feature added : 
Startup script execution 

Note: Startup script automatically execute(once) on Node.js server start ,even cluster module is enabled script will load only once.

**Comment by jayesh-sapkale** [comment]
Log Entry : 
4-August-2016 +0216

Jswalker Core and supporting libs were moved from github to Gitlab Version Control system.
  - Core moved to gitlab
  - Project under jswalker moved to gitlab
  - Procedure and Documentation for jswalker took time

Outcome : Test result postpone till 7-August-2016  +1900
